pkg/arvo/app/launch.hoon

```diff
@@ -162,6 +162,8 @@
   |=  =path
   ^-  (unit (unit cage))
   ?+  path  (on-peek:def path)
```

Comment:
Consider producing `[~ ~]` (for "invalid request") here, as is done in s3-store.

Also, seems the `team:title` check was missing here? Worth adding.